### ISSUE TITLE
Inline Link not rendered as a clickable link

### ISSUE THREAD

**Issue by aaronjedwards** [diegomura/react-pdf#1517]
**Describe the bug**
When rendering a `Link` inline (wrapped in a `Text` component), the resulting PDF does not render the link as clickable.

**To Reproduce**
Simply wrap a `Link` within a `Text` component:

```jsx
<Text>
  This has a
  <Link src={'https://google.com'}>
   <Text>Link </Text>
 </Link>
  inline, but it isn't clickable, likely due somehow to being rendered in a {"<Text>"} component...
</Text>
```

Here is [an example of this issue](https://react-pdf.org/repl?code=3187b0760ce02e00408a057025803c450298c0bc300500943807cf805030c00f0022230080b6198531165d400a02180e659a004f003619b006f61622003a0046200099080beed3a72a005430a361d36500122003b8c603cc0c24300138b2519ecc1e86b407a5dfa351aa0032486000d63010f6c09200e400165050000e10005c9e9e7c20207c62b2a08cd1ea1e9a00903a7a6c41a1d4de157ea5549e5521f55a3e6cc122c11800fc8dedad94e5bec59cdab1481030b13c53ee465acde19131f149a9e999d9b9f985839c65034bfd75a3654dc12da3949ddd003430f208b0482f1060d1b0c05dc0213cf26207974421811108604a0420840cc58b9860501023c30c13e0389c2e0c1296c361e0c0240022619b009aa4b0c312e01614164b4d189c469a46af0047e469d018cc563b008006e321900062e01a638f8934c3d8f0120e000cc788c24182523068801e420661e08894d13b87022c0155c412c934a796522b91f1a03c28121807918678209e1026bb54a4f000dc008c00664f0013400fada0034800d4006c200007001d440def90015494fec6086001a004e1e001040bb204acba264551f205a0482c1a41829ae000ca50510609bb10c06069c0471da30d28e228542a99669123c2512951da1022455fe802b3ebc793e9d80f8002174223182ae8f2e3c13a9ea34cf624000bc45da85e1f28aa15c235e6251c792dac26e5e302a8013000591f4a1307d00b2e8f83018d601a9170f537c454141525484634352d47572def47c78678e17b15f4d1df2813f0bdbf1817d1fd008442a5029070320e83ec58334460787b1c5301b704861154ff000191f07c0d041e47b4a017df13823f2fc555f4e34a258b638219228c92a004315655555433d0c260412808a9f0ce014f6394ca308e2348df4008f180a8068ba355000ac10680905948426338423d4a438d6d09066020001680025185ac5d3f4db0588108cca04ce0923171ed2b044192ef63358f62cf4bdaf74ac8de3f88e1a2ced27171e29802ce92c895398ecb824e3775fd8acd0ec87220d54a0d6060ca340110403c3553e11c4f204c7c270100039261e40aa24f1ddd578907018d7f82010044678302f3286aa48b323c450b8bdc607f55ace0c4594a01542eca1cf3e1e25bb28f6ac0ceba26eb255dbc901a86e88468c0c692bf54adf9321428c078600a02e06841564470c0670a52a110541d02c13c620ab200) that I reproduced with the REPL.

**Expected behavior**
I would expect the `Link` to be properly rendered as clickable in the resulting PDF document.

Is there a documented way of properly rendering a `Link` component inline that I am unaware of? Im happy to look into resolving this as well, just not exactly sure where to start.

**Comment by jeetiss** [comment]
Thanks for the issue
There is a fix #1503
It isn't released yet, so bug is reproducible in the current version

**Comment by diegomura** [comment]
Fixed and merged
